pyproject.toml

```diff
@@ -74,10 +74,33 @@ packages = ["timml", "timml.besselaesnumba"]
 [tool.setuptools.dynamic]
 version = { attr = "timml.version.__version__" }
 
-[tool.black]
-line-length = 88
+[tool.ruff]
+exclude = ["examples"]
```

Comment:
Why ignore examples? We can also run ruff on the example notebooks.

Reply:
Good point, I thought I was getting a lot of complaints here, but they're quite manageable with the current set of rule ignores. I'll make another commit.